Makefile.am

```diff
@@ -1,5 +1,7 @@
+if HAVE_EMBEDDED_YAJL
 DIST_SUBDIRS = yajl
 SUBDIRS = yajl
+endif
```

Comment:
it is causing the CI to fail.

Does it work better if you protect `DIST_SUBDIRS` only?

Reply:
Actually, it seemed to work when I protected `SUBDIRS` only instead. I think that behavior makes sense to me, but I'm not 100% sure.